src/main/menu.js

```diff
@@ -12,7 +12,7 @@ const getMenuTemplate = (mainWindow) => {
                 {
                     label: "About", role: 'about',
                     click() {
-                        const win = new electron.BrowserWindow({
+                        let win = new electron.BrowserWindow({
```

Comment:
There's nothing that requires this to be a `let` - we should prefer `const` declarations where possible.

Reply:
@dfcowell I know but `win = null` when closing the window requires it to be a let.  Cheers

Reply:
D'oh! Didn't see that.